plumbing/object/tree_test.go

```diff
@@ -48,6 +48,22 @@ func (s *TreeSuite) TestType(c *C) {
 	c.Assert(s.Tree.Type(), Equals, plumbing.TreeObject)
 }
 
+func (s *TreeSuite) TestTree(c *C) {
```

Comment:
Good catch.
Fixed in a new commit.
Also add the equivalent test for `File`, for consistency.